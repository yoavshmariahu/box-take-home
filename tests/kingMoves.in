k a1
K e5

[G S B R P]
[p r b s g]

move a1 a2
move e5 d4
move a2 b2
move d4 e4
move b2 c3
move e4 e5
move c3 c2
move e5 e4
move c2 b3
move e4 d4
move b3 a3
move d4 e3
move a3 b2
move e3 d4
move b2 a1
move d4 e5
