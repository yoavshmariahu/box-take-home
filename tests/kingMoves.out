UPPER player action: move d4 e5
5 |__|__|__|__| K|
4 |__|__|__|__|__|
3 |__|__|__|__|__|
2 |__|__|__|__|__|
1 | k|__|__|__|__|
    a  b  c  d  e

Captures UPPER: G S B R P
Captures lower: p r b s g

lower> 
